Appraisals

```diff
@@ -0,0 +1,15 @@
+appraise "rails-4-0" do
```

Comment:
Could you remove appraisal? It is not necessary and I prefer to keep our setup simple.